mautic/mautic
# [UI] Define the right cursor for interface elements

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR makes:
- Page headings and other interface text use the default cursor instead of text cursor, inheriting from body
- Everything that is in a field use the text cursor
- All clickable things use the pointer cursor

The overall idea is to improve the interaction feedback






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts
3. Hover over the heading (normal cursor)
4. Create a new Contact > Fill the fields and hover over the text inside the fields (will display the text cursor)
5. Hover over the Saving & Close to see the pointer cursor
